CHANGELOG.md

```diff
@@ -18,6 +18,8 @@
 - [#2338](https://github.com/influxdb/influxdb/pull/2338): Fix panic if tag key isn't double quoted when it should have been
 - [#2340](https://github.com/influxdb/influxdb/pull/2340): Fix SHOW DIAGNOSTICS panic if any shard was non-local.
 - [#2351](https://github.com/influxdb/influxdb/pull/2351): Fix data race by rlocking shard during diagnostics.
+- [#2350](https://github.com/influxdb/influxdb/pull/2350): Issue fix for :influxd -hostname localhost.
```

Comment:
#2350 is an issue, not a PR. The usual style is not to add issue URL, but just the PR URL, and say something like "fixes 2350". Unless I am missing something?